notebooks/demo2/transformer-relevance-sparse.ipynb

```diff
@@ -0,0 +1,2042 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    model_name_4_S3 = ['pruned_RELEVANCE1',
</pre><p>Could we rename this to distilbert_mnli_pruned80, distilbert_qqp_pruned80, obert_mnli_pruned90 both in model_name_4_S3 and in the paths (instead of sparse1 sparse2 sparse3 )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/234/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>